.github/workflows/openmdao_test_workflow.yml

```diff
@@ -191,23 +191,54 @@ jobs:
           pip install git+https://github.com/swryan/coveralls-python
           SITE_DIR=`python -c 'import site; print(site.getsitepackages()[-1])'`
           coveralls --base_dir $SITE_DIR
-          
+
       - name: Make docs
         shell: bash -l {0}
         run: |
           if [[ "${{ matrix.PETSc }}" && "${{ matrix.OPTIONAL }}" == "[all]" ]]; then
+
+            cd openmdao/docs
+            if [[ "${{ secrets.SNOPT_LOCATION_71 }}" || "${{ secrets.SNOPT_LOCATION_77 }}" ]]; then
```

Comment:
`SNOPT_LOCATION_71` should be `SNOPT_LOCATION_72`

Reply:
Fixed